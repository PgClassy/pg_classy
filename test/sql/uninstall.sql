\set ECHO none
\i test/pgxntool/setup.sql

DROP EXTENSION classy;
