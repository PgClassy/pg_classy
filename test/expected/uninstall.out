\set ECHO none

psql:test/pgxntool/tap_setup.sql:3: NOTICE:  42P06: schema "tap" already exists, skipping
LOCATION:  CreateSchemaCommand, schemacmds.c:115
psql:test/pgxntool/tap_setup.sql:5: NOTICE:  42710: extension "pgtap" already exists, skipping
LOCATION:  CreateExtension, extension.c:1535
